src/DatePicker/props.ts

```diff
@@ -0,0 +1,51 @@
+import { DayPickerInputProps } from 'react-day-picker';
+
+import { InputProps } from '../Input';
+
+type BlendedProps = InputProps & DayPickerInputProps;
+
+export type Props = Partial<BlendedProps> & {
+  /**
+   * Set className
+   *
+   * @default ''
+   **/
+  className?: string;
```

Comment:
The propType column is showing `unknown` for all of these types. Do you know why?